### PR TITLE
finished code challenge

### PR DESCRIPTION
**Pull request by stacyburris**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: javascript/code-challenges/treeintersection/tree-intersection.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    };
+    preOrderHelperTwo(root);
+  }
+  preOrderTwo(treeTwo.root);
+
+  return outputArray;
+}
+
+
+module.exports = treeIntersection;
```
Comment (adrianhuebner) [inline javascript/code-challenges/treeintersection/tree-intersection.js:41]: Looks really great, just would like to see testing added!
